.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: required
```

Comment:
Snuck in this cleanup, this is deprecated according to the warning in Travis.

<img width="497" alt="Screen Shot 2020-06-26 at 4 24 25 PM" src="https://user-images.githubusercontent.com/1813380/85903150-44e00e80-b7cb-11ea-97aa-aec91490d02b.png">